.github/workflows/tests.yml

```diff
@@ -0,0 +1,72 @@
+name: tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+    type: [ "opened", "reopened", "synchronize" ]
+  schedule:
+    - cron: '0 12 * * 0'  # run once a week on Sunday
+
+jobs:
+  tests:
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: P0sTGrE5*Pa55W0rt!
+        # Set health checks to wait until postgres has started
+        options: >-
```

Comment:
So the health checks were needed.

Reply:
Not sure, I don't know what I am doing here.